### PR TITLE
Remove broken and inactive links, fix redirected links, and add a link to the community calendar

### PR DESCRIPTION
**Pull request by 2color**
## What's in this PR

- Remove broken and inactive links
- Fix some spelling mistakes
- Fix redirected links
- Add a community calendar section to the community page as per #1237

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

File: docs/concepts/usage-ideas-examples.md [hunk 1]
```diff
@@ -142,7 +132,7 @@ If you are currently using MongoDB, you might be more comfortable with [ThreadDB
 
 ### IPFS hosting with Textile
 
-[Textile](https://Textile.io) is a hosting company on IPFS developing the layers on top of IPFS. Among other things, they came up with separated _cloud_ environments called [buckets](https://docs.textile.io/hub/buckets/). Textile has also built an impressive toolset for [building decentralized applications and integrations](https://blog.textile.io/announcing-the-textile-protocol-hub/).
+[Textile](https://linktr.ee/textileio) is a hosting company on IPFS developing the layers on top of IPFS. Among other things, they came up with separated _cloud_ environments called [buckets](https://docs.textile.io/hub/buckets/). Textile has also built an impressive toolset for [building decentralized applications and integrations](https://blog.textile.io/announcing-the-textile-protocol-hub/).
```
Comment (DannyS03) [inline docs/concepts/usage-ideas-examples.md:135]:
```suggestion
[Textile](https://linktr.ee/textileio) is developing the layers on top of IPFS to extend its hosting capabilities. Among other things, they came up with separated _cloud_ environments called [buckets](https://docs.textile.io/hub/buckets/). Textile has also built an impressive toolset for [building decentralized applications and integrations](https://blog.textile.io/announcing-the-textile-protocol-hub/).
```

File: docs/community/README.md [hunk 2]
```diff
@@ -23,6 +23,14 @@ The official [IPFS forums](https://discuss.ipfs.io/) are the primary home of IPF
 
 Casual chat fan? Here are [all our channels](chat.md). There's getting-started advice for you here, too.
 
+## Community Calendar
+
+If you're an IPFS implementer, consumer, or user you are welcome to join the community meetings we host regularly.
+
+Stay up to date by subscribing to the [**IPFS Community Calendar**](https://calendar.google.com/calendar/embed?src=ipfs.io_eal36ugu5e75s207gfjcu0ae84%40group.calendar.google.com) and joining community meetings such as triage sessions, the IPFS implementers sync, and the IPLD sync.
```
Comment (DannyS03) [inline docs/community/README.md:30]:
```suggestion
Stay up-to-date by subscribing to the [**IPFS Community Calendar**](https://calendar.google.com/calendar/embed?src=ipfs.io_eal36ugu5e75s207gfjcu0ae84%40group.calendar.google.com) and joining community meetings such as triage sessions, the IPFS implementers sync, and the IPLD sync.
```

**Review by DannyS03 — CHANGES_REQUESTED** [review]
thanks @2color, added some suggestions

File: docs/concepts/persistence.md [hunk 3]
```diff
@@ -74,12 +74,13 @@ To ensure that your important data is retained, you may want to use a pinning se
 
 Some available pinning service providers are:
 
-- [Axel](https://www.axel.org/2019/07/23/qa-with-the-developers-of-axel-ipfs/)
-- [Eternum](https://www.eternum.io/)
+- [Web3.Storage](https://web3.storage/)
```
Comment (mishmosh) [inline docs/concepts/persistence.md:77]: Should we put this list in alphabetical order?

File: docs/concepts/usage-ideas-examples.md [hunk 4]
```diff
@@ -14,21 +14,19 @@ One of the simplest use-cases to grasp is the sharing of files between peers.
 
 ### Desktop applications
 
-[IPFS Desktop](https://github.com/ipfs-shipyard/ipfs-desktop) is the official desktop client for IPFS. It comes with a built-in IPFS node that lets you pin files and gives you a link to share them. This is generally seen as the easiest point-of-entry into the IPFS ecosystem.
+[IPFS Desktop](https://github.com/ipfs/ipfs-desktop) is the official desktop client for IPFS. It comes with a built-in IPFS node that lets you pin files and gives you a link to share them. This is generally seen as the easiest point of entry into the IPFS ecosystem.
```
Comment (mishmosh) [inline docs/concepts/usage-ideas-examples.md:17]:
```suggestion
[IPFS Desktop](https://github.com/ipfs/ipfs-desktop) is the official desktop client for IPFS. It comes with a built-in IPFS node that lets you pin files and gives you a link to share them. This is one of the easiest ways to start pinning your files to IPFS.
```
simplify

**Review by mishmosh — APPROVED** [review]
Thanks for the cleanup <3. Left a couple minor suggestions, but looks good.

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

File: docs/concepts/persistence.md (hunk 3 above)
Comment (2color) [reply docs/concepts/persistence.md:77]: We probably could. In this first run, I thought to put the ones that have full IPFS compatibility first (including DHT providing). 

As it turns out Axel is not even offering IPFS anymore.

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Review by DannyS03 — APPROVED** [review]

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

File: docs/concepts/usage-ideas-examples.md [hunk 5]
```diff
@@ -116,7 +110,7 @@ The developers of the [Nix package manager](https://nixos.org/) are working on i
 
 ### Hosting software containers
 
-Netflix is on its way to use IPFS to [synchronize their Docker containers worldwide](https://blog.ipfs.tech/2020-02-14-improved-bitswap-for-container-distribution/). Since each node fetches them from the fastest peers they know, synchronization is substantially faster than regular sync methods.
+Netflix is on its way to using IPFS to [synchronize their Docker containers worldwide](https://blog.ipfs.io/2020-02-14-improved-bitswap-for-container-distribution/). Since each node fetches them from the fastest peers they know, synchronization is substantially faster than traditional synchronization methods.
```
Comment (lidel) [inline docs/concepts/usage-ideas-examples.md:113]: @DannyS03 in future edits, use .tech.

We moved blog, docs, and main website  to https://blog.ipfs.tech  :-)  and want .io to be only used for gateway at `ipfs.io/ipfs`

Comment (2color) [reply docs/concepts/usage-ideas-examples.md:113]: Fixed in 06b83516c7f81c391800f65321e9ce259a3c6742

**Pull request merged**